tasks/build.yml

```diff
@@ -64,10 +64,9 @@
     mode: 0775
     src: prometheus_node_exporter.sysvinit.j2
 
-- name: Make Prometheus Node Exporter service start on boot
+- name: Prometheus Node Exporter service start
   become: yes
   service:
     name: prometheus-node-exporter
-    enabled: yes
+    enabled: "{{ prometheus_node_exporter.start_on_boot }}"
     state: started
```

Comment:
Do we need to actually start the service here? Can we just enable/disable it? I think we should a separate task in configure.yml that actually starts the service if prometheus_node_exporter.start_on_boot is true.